rules/windows/persistence_run_key_and_startup_broad.toml

```diff
@@ -11,7 +11,7 @@ Identifies run key or startup key registry modifications. In order to survive re
 attackers will modify run keys within the registry or leverage startup folder items as a form of persistence.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support